docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md

```diff
@@ -0,0 +1,32 @@
+# Restart jobs in other regions shouldn't work for run with cloud dependent parameters
+
+Test verifies that restart jobs in other regions shouldn't work for run with cloud dependent parameters.
+
+**Prerequisites**:
+- Admin user
+- Perform [3122_1](3122_1.md) case
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Create object storage
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the Tools page | |
+| 3 | Select test tool | |
+| 4 | At the tool page, hover over the **Run v** button | |
+| 5 | Click the **Custom settings** button in the list | |
+| 6 | Expand the **Exec environment** section | |
+| 7 | Select `p2.8xlarge` value for **Node type** | |
```

Comment:
I suggest not to specify node type explicitly here. And change the step to something like:
"Select the **Node type** from the prerequisites"

And specify in the Prerequisites:
"- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region"

Reply:
Done